src/main/java/tech/gruppone/stalker/server/controllers/PasswordController.java

```diff
@@ -23,11 +24,11 @@
   PasswordService passwordService;
   UserService userService;
 
-  // TODO should throw InvalidEmailException!
   @PostMapping("/user/password/recovery")
   @ResponseStatus(HttpStatus.NO_CONTENT)
   public Mono<Void> postUserPasswordRecovery(
-      @RequestBody final PostUserPasswordRecoveryRequestBody requestBody) {
+      @RequestBody final PostUserPasswordRecoveryRequestBody requestBody)
+      throws NoSuchAlgorithmException {
```

Comment:
il throw l'avevo tolto perché non è corretto che l'endpoint throwi questo errore.. java vuole che (dato che questo errore non si verifica mai) venga convertito in un eccezione per cui non serva aggiungere throws (tipo una specializzazione di runtimeexception). comunque non ti preoccupare, faccio io

Reply:
Ok. Comunque l'ho aggiunto nuovamente perchè rimuovendolo viene segnalato un errore proprio a riga 30